### PR TITLE
Added Slack to blockable services

### PR DESCRIPTION
**Pull request by sandervankasteel**

### CODE REVIEW

File: services/slack.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
...esentation"><g fill="none" fill-rule="evenodd"><path d="M19.712.133a5.381 5.381 0 0 0-5.376 5.387 5.381 5.381 0 0 0 5.376 5.386h5.376V5.52A5.381 5.381 0 0 0 19.712.133m0 14.365H5.376A5.381 5.381 0 0 0 0 19.884a5.381 5.381 0 0 0 5.376 5.387h14.336a5.381 5.381 0 0 0 5.376-5.387 5.381 5.381 0 0 0-5.376-5.386" fill="#44BEDF"></path><path d="M53.76 19.884a5.381 5.381 0 0 0-5.376-5.386 5.381 5.381 0 0 0-5.376 5.386v5.387h5.376a5.381 5.381 0 0 0 5.376-5.387m-14.336 0V5.52A5.381 5.381 0 0 0 34.048.133a5.381 5.381 0 0 0-5.376 5.387v14.364a5.381 5.381 0 0 0 5.376 5.387 5.381 5.381 0 0 0 5.376-5.387" fill="#2EB67D"></path><path d="M34.048 54a5.381 5.381 0 0 0 5.376-5.387 5.381 5.381 0 0 0-5.376-5.386h-5.376v5.386A5.381 5.381 0 0 0 34.048 54m0-14.365h14.336a5.381 5.381 0 0 0 5.376-5.386 5.381 5.381 0 0 0-5.376-5.387H34.048a5.381 5.381 0 0 0-5.376 5.387 5.381 5.381 0 0 0 5.376 5.386" fill="#ECB22E"></path><path d="M0 34.249a5.381 5.381 0 0 0 5.376 5.386 5.381 5.381 0 0 0 5.376-5.386v-5.387H5.376A5.381 5.381 0 0 0 0 34.25m14.336-.001v14.364A5.381 5.381 0 0 0 19.712 54a5.381 5.381 0 0 0 5.376-5.387V34.25a5.381 5.381 0 0 0-5.376-5.387 5.381 5.381 0 0 0-5.376 5.387" fill="#E01E5A"></path></g></svg>
```
Comment (Sergey-Lyapin) [inline services/slack.yml:9]: Hello @sandervankasteel 
SVG icon must be black and white, please use this one:
```
<svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 54 54"><path fill-rule="evenodd" d="M19.71.13a5.38 5.38 0 0 0-5.37 5.39 5.38 5.38 0 0 0 5.37 5.39h5.38V5.52A5.38 5.38 0 0 0 19.7.13m0 14.37H5.38A5.38 5.38 0 0 0 0 19.88a5.38 5.38 0 0 0 5.38 5.4H19.7a5.38 5.38 0 0 0 5.38-5.4 5.38 5.38 0 0 0-5.38-5.38m34.06 5.38a5.38 5.38 0 0 0-5.38-5.38 5.38 5.38 0 0 0-5.37 5.38v5.4h5.37a5.38 5.38 0 0 0 5.38-5.4m-14.34 0V5.52A5.38 5.38 0 0 0 34.05.13a5.38 5.38 0 0 0-5.38 5.39v14.36a5.38 5.38 0 0 0 5.38 5.4 5.38 5.38 0 0 0 5.37-5.4M34.05 54a5.38 5.38 0 0 0 5.37-5.39 5.38 5.38 0 0 0-5.37-5.38h-5.38v5.38A5.38 5.38 0 0 0 34.05 54m0-14.37h14.33a5.38 5.38 0 0 0 5.38-5.38 5.38 5.38 0 0 0-5.38-5.39H34.05a5.38 5.38 0 0 0-5.38 5.39 5.38 5.38 0 0 0 5.38 5.38M0 34.25a5.38 5.38 0 0 0 5.38 5.39 5.38 5.38 0 0 0 5.37-5.4v-5.38H5.38A5.38 5.38 0 0 0 0 34.25m14.34 0V48.6A5.38 5.38 0 0 0 19.7 54a5.38 5.38 0 0 0 5.38-5.39V34.25a5.38 5.38 0 0 0-5.38-5.39 5.38 5.38 0 0 0-5.37 5.39"/></svg>
```

File: services/slack.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+id: slack
+name: Slack
+rules:
+  - "||slack.com^"
```
Comment (Sergey-Lyapin) [inline services/slack.yml:4]: Quotes must be single.

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: services/slack.yml (hunk 1 above)
Comment (sandervankasteel) [reply services/slack.yml]: Done!

File: services/slack.yml (hunk 2 above)
Comment (sandervankasteel) [reply services/slack.yml]: Updated :)

**Review by Sergey-Lyapin — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**